Emby.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -569,56 +563,51 @@ public string EncodeImage(string inputPath, DateTime dateModified, string output
-                        using (var canvas = new SKCanvas(saveBitmap))
+                        // set background color if present
+                        if (hasBackgroundColor)
                         {
-                            // set background color if present
-                            if (hasBackgroundColor)
-                            {
-                                canvas.Clear(SKColor.Parse(options.BackgroundColor));
-                            }
+                            canvas.Clear(SKColor.Parse(options.BackgroundColor));
+                        }
 
-                            // Add blur if option is present
-                            if (blur > 0)
-                            {
-                                using (var paint = new SKPaint())
-                                {
-                                    // create image from resized bitmap to apply blur
-                                    using (var filter = SKImageFilter.CreateBlur(blur, blur))
-                                    {
-                                        paint.ImageFilter = filter;
-                                        canvas.DrawBitmap(resizedBitmap, SKRect.Create(width, height), paint);
-                                    }
-                                }
-                            }
-                            else
+                        // Add blur if option is present
+                        if (blur > 0)
+                        {
+                            // create image from resized bitmap to apply blur
+                            using (var paint = new SKPaint())
+                            using (var filter = SKImageFilter.CreateBlur(blur, blur))
                             {
-                                // draw resized bitmap onto canvas
-                                canvas.DrawBitmap(resizedBitmap, SKRect.Create(width, height));
+                                paint.ImageFilter = filter;
+                                canvas.DrawBitmap(resizedBitmap, SKRect.Create(width, height), paint);
                             }
+                        }
+                        else
+                        {
+                            // draw resized bitmap onto canvas
+                            canvas.DrawBitmap(resizedBitmap, SKRect.Create(width, height));
+                        }
 
-                            // If foreground layer present then draw
-                            if (hasForegroundColor)
-                            {
-                                Double opacity;
-                                if (!Double.TryParse(options.ForegroundLayer, out opacity)) opacity = .4;
+                        // If foreground layer present then draw
+                        if (hasForegroundColor)
+                        {
+                            Double opacity;
+                            if (!Double.TryParse(options.ForegroundLayer, out opacity)) opacity = .4;
```

Comment:
I think at least splitting this onto a new line, if not adding `{...}` makes it more readable.